main/src/main/java/sbt/internal/ReverseLookupClassLoader.java

```diff
@@ -8,7 +8,9 @@
 package sbt.internal;
 
 import java.io.File;
+import java.io.IOException;
```

Comment:
I think the changes to ReverseLookupClassLoader were actually good and ideally wouldn't be reverted. Maybe I should take a stab at reverting this so that the baby doesn't get thrown out with the bathwater.